Content Assist Broken

With current master and xtext 2.14 the code completion is broken. 
Steps to reproduce:

within a function, type msg. ctrl-space
The following exception is thrown:

Caused by: java.lang.IllegalStateException
	at org.eclipse.xtext.parser.antlr.UnorderedGroupHelper$State.canLeave(UnorderedGroupHelper.java:237)
	at org.eclipse.xtext.parser.antlr.UnorderedGroupHelper.canLeave(UnorderedGroupHelper.java:153)
	at org.eclipse.xtext.ide.editor.contentassist.antlr.IgnoreFirstEntranceUnorderedGroupHelper.canLeave(IgnoreFirstEntranceUnorderedGroupHelper.java:70)
	at org.eclipse.xtext.ide.editor.contentassist.antlr.EofListener.announceEof(EofListener.java:60)
	at org.eclipse.xtext.ide.editor.contentassist.antlr.ObservableXtextTokenStream.LA(ObservableXtextTokenStream.java:48)
	at org.antlr.runtime.DFA.predict(DFA.java:97)
	at org.eclipse.xtext.ide.editor.contentassist.antlr.internal.DFA.predict(DFA.java:26)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__SimpleStatement__Alternatives(InternalSolidityParser.java:10343)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.ruleSimpleStatement(InternalSolidityParser.java:3231)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__Statement__Alternatives_5_0(InternalSolidityParser.java:10281)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__Statement__Group_5__0__Impl(InternalSolidityParser.java:28908)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__Statement__Group_5__0(InternalSolidityParser.java:28861)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__Statement__Alternatives(InternalSolidityParser.java:10051)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.ruleStatement(InternalSolidityParser.java:3061)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__Block__StatementsAssignment_2(InternalSolidityParser.java:53614)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__Block__Group__2__Impl(InternalSolidityParser.java:23218)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__Block__Group__2(InternalSolidityParser.java:23159)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__Block__Group__1(InternalSolidityParser.java:23085)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__Block__Group__0(InternalSolidityParser.java:23006)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.ruleBlock(InternalSolidityParser.java:2041)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__FunctionDefinition__BlockAssignment_8_1(InternalSolidityParser.java:54363)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__FunctionDefinition__Alternatives_8(InternalSolidityParser.java:9729)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__FunctionDefinition__Group__8__Impl(InternalSolidityParser.java:26183)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__FunctionDefinition__Group__8(InternalSolidityParser.java:26141)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__FunctionDefinition__Group__7(InternalSolidityParser.java:26046)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__FunctionDefinition__Group__6(InternalSolidityParser.java:25939)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__FunctionDefinition__Group__5(InternalSolidityParser.java:25860)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__FunctionDefinition__Group__4(InternalSolidityParser.java:25760)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__FunctionDefinition__Group__3(InternalSolidityParser.java:25681)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__FunctionDefinition__Group__2(InternalSolidityParser.java:25581)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__FunctionDefinition__Group__1(InternalSolidityParser.java:25502)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__FunctionDefinition__Group__0(InternalSolidityParser.java:25423)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.ruleFunctionDefinition(InternalSolidityParser.java:2551)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__ContractPart__Alternatives(InternalSolidityParser.java:9094)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.ruleContractPart(InternalSolidityParser.java:936)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__ContractDefinition__FeaturesAssignment_5(InternalSolidityParser.java:52340)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__ContractDefinition__Group__5__Impl(InternalSolidityParser.java:16096)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__ContractDefinition__Group__5(InternalSolidityParser.java:16037)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__ContractDefinition__Group__4(InternalSolidityParser.java:15963)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__ContractDefinition__Group__3(InternalSolidityParser.java:15863)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__ContractDefinition__Group__2(InternalSolidityParser.java:15774)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__ContractDefinition__Group__1(InternalSolidityParser.java:15685)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__ContractDefinition__Group__0(InternalSolidityParser.java:15606)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.ruleContractDefinition(InternalSolidityParser.java:766)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__SourceUnit__MemberAssignment_1_2(InternalSolidityParser.java:51841)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__SourceUnit__UnorderedGroup_1__Impl(InternalSolidityParser.java:51109)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__SourceUnit__UnorderedGroup_1__1(InternalSolidityParser.java:51263)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.synpred192_InternalSolidity_fragment(InternalSolidityParser.java:59210)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.synpred192_InternalSolidity(InternalSolidityParser.java:59415)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__SourceUnit__UnorderedGroup_1__0(InternalSolidityParser.java:51194)
	at com.yakindu.solidity.ide.contentassist.antlr.internal.InternalSolidityParser.rule__SourceUnit__UnorderedGroup_1(InternalSolidityParser.java:50804)
	... 21 more

--- a/plugins/com.yakindu.solidity/src/com/yakindu/solidity/Solidity.xtext
+++ b/plugins/com.yakindu.solidity/src/com/yakindu/solidity/Solidity.xtext
@@ -31,8 +31,8 @@ SolidityModel:
 
 SourceUnit returns types::Package:
 	{SourceUnit}
-	((pragma+=PragmaDirective)* &
-	(imports+=ImportDirective)* &
+	((pragma+=PragmaDirective)* 
+	(imports+=ImportDirective)* 
 	member+=ContractDefinition);
 
 PragmaDirective returns types::PackageMember:

